build/ToolsetPackages/project.lock.json

```diff
@@ -29,12 +29,12 @@
           "lib/net45/Microsoft.CodeAnalysis.Test.Resources.Proprietary.dll": {}
         }
       },
-      "Microsoft.DiaSymReader/1.0.5": {
+      "Microsoft.DiaSymReader/1.0.5.1": {
         "compile": {
-          "lib/net45/Microsoft.DiaSymReader.dll": {}
+          "lib/net20/Microsoft.DiaSymReader.dll": {}
```

Comment:
The package has two targets:
net20
portable-net45+win8

I'm not sure why net20 was picked.